Frameworks/CoreFoundation/URL.subproj/CFURL.c

```diff
@@ -1407,14 +1416,16 @@ CF_EXPORT CFStringRef CFURLCreateStringByAddingPercentEscapes(CFAllocatorRef all
                         // surrogate pair conversion failed
                         break;
                     }
-                } else {
+                }
```

Comment:
Formatter messed things up somehow. Fixed.

---
In reply to: [77075978](https://github.com/Microsoft/WinObjC/pull/946#discussion_r77075978) [](ancestors = 77075978)